content/departments/cloud/trial_mi.md

```diff
@@ -121,10 +123,10 @@ When trial expires and customer do not wish to sign the deal, instance requestor
 4. Finalisation (Cloud Team member)
 
    - checkout `<CUSTOMER>/create-instance` branch in`deploy-sourcegraph-managed` repository
+   - `cd <CUSTOMER>`
    - need to set the license on the instance (the license key should be added to the issue, unless it is default PLG licence)
      - run `mi set-license -license-key "$LICENSE_KEY"` (for PLG trials flag `--plg-default` instead of `-license-key` should be used - will use shared PLG licence key)
    - obtain customer reset link via `mi reset-customer-password --email <customer admin email>` and paste it into the GitHub issue
-   - when [giving customer access](./technical-docs/v1.1/mi1-1_creation_process.md#giving-customer-access) is done via comment in New Trial Managed Instance request issue, alert in `#cloud-notifications` should be closed.
```

Comment:
Something I've wondered about this step - do we close the issue at this point? IMO we should, otherwise we just have lots of open issues that appear open but are actually done

Reply:
> Something I've wondered about this step - do we close the issue at this point? IMO we should, otherwise we just have lots of open issues that appear open but are actually done

Make sense, will add point to close the issue, b/c all work is done on our side.